providers/1_8Provider/src/main/java/net/ess3/provider/providers/LegacyBiomeNameProvider.java

```diff
@@ -0,0 +1,15 @@
+package net.ess3.provider.providers;
+
+import net.ess3.provider.BiomeNameProvider;
+import net.essentialsx.providers.ProviderData;
+import org.bukkit.block.Block;
+
+import java.util.Locale;
+
+@ProviderData(description = "Legacy Item Unbreakable Provider")
+public class LegacyBiomeNameProvider implements BiomeNameProvider {
+    @Override
+    public String getBiomeName(final Block block) {
+        return block.getBiome().name().toLowerCase(Locale.ENGLISH);
```

Comment:
Add a comment explaining what this is doing - compiling against an enum on 1.8 and relying on a Commodore rewrite / bytecode magic / ??? to work on latest

Reply:
```suggestion
        // For some reason, compiling against modern versions causes this call to break, possibly related to OldEnum?
        // Compiling this against versions that still have proper enums allow this to work
        return block.getBiome().name().toLowerCase(Locale.ENGLISH);
```